server/controllers/match/placementRequestsController.test.ts

```diff
@@ -84,7 +84,7 @@ describe('PlacementRequestsController', () => {
       expect(getPaginationDetails).toHaveBeenCalledWith(request, paginationDetails.hrefPrefix)
```

Comment:
Would be useful to update the first commit message so that the title describes the change rather than the ticket, and then in the body you can explain how this achieves that/what the cause of the issue was. Something like this:

```
Fix `taskType` query param

Placement requests were showing up on the tasks screens. We were
using `TaskType` as a query param rather than `Array<TaskType>`.
This fixes the typing of the query param and specifies the task types
we want to see on the relevant screens
```

Reply:
The commit message still needs revising ([see here for suggestion](https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/1531/files#r1504206883))